.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: python
 python:
-  - 2.7
   - 3.6
```

Comment:
If we're going for Python 3, should we also go for a non-deprecated 3 version? Or are you planning to tackle that in a follow-on PR?

Reply:
Yea I agree, just wasn't sure if CI is still running (or honestly if the project is still being maintained)

Would it make sense to just migrate to github actions? Can do that here or in a separate PR (whichever you prefer)